src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -268,7 +270,7 @@ class Liberty implements Plugin<Project> {
     }
 
     public static void checkEtcServerEnvProperties(Project project) {
-        if (project.liberty.outputDir == null) {
+        if (project.liberty.server.outputDir == null) {
```

Comment:
We're treating this as a broader liberty level property rather than a server level property, so I think it's best to keep this as `project.liberty.outputDir`. If the user sets an outputDir at the server level it will override whatever is set at the liberty level.